app/src/main/java/cm/aptoide/pt/search/view/SearchResultAdapter.java

```diff
@@ -27,6 +28,7 @@
   private boolean adsLoaded = false;
   private boolean isLoadingMore = false;
   private CrashReport crashReport;
+  private DecimalFormat oneDecimalFormatter = new DecimalFormat("#.##");
```

Comment:
you should probably pass this in the constructor since we're doing that pretty much everywhere else